setup/android_build_env.sh

```diff
@@ -14,6 +14,7 @@ DEBIAN_11_PACKAGES="libncurses5"
 PACKAGES=""
 
 echo "Adding GitHub apt key and repository!"
+sudo apt-get install software-properties-common -y
```

Comment:
Just `apt`, remove the `-get` to keep it consistent with the rest of the apt installs we do in here.